tests/compiler/test_index.py

```diff
@@ -50,39 +50,28 @@ def test_issue_121_non_matching_index(snippetcompiler):
 def test_issue_122_index_inheritance(snippetcompiler):
```

Comment:
This is no longer related to https://github.com/inmanta/inmanta-core/issues/122


it should fail with

```
E               inmanta.ast.statements.generator.IndexAttributeMissingInConstructorException: Invalid Constructor call:
E               	* Missing attribute 'path'. The attribute __config__::Repository.path is part of an index. (reported in Construct(Repository) (/tmp/tmp268dmcty/main.cf:25))
```

But it fails with double set

Reply:
https://github.com/inmanta/inmanta-core/pull/7559#discussion_r1580692161 but I'll have another look I guess

Reply:
The intent of this testcase is to set the value of an index attribute (the index is on the parent) in an implementation of the child, so e.g. set 'name' in an implementation